vunit/ui.py

```diff
@@ -393,6 +394,36 @@ def add_external_library(self, library_name, path, vhdl_standard=None):
         self._project.add_library(library_name, abspath(path), vhdl_standard, is_external=True)
         return self.library(library_name)
 
+    def add_csv(self, project_csv_path, vhdl_standard=None):
+        """
+        Add a project configuration, mapping all the libraries and files
+        :param project_csv_path: path to csv project spec
+        :param vhdl_standard: The VHDL standard used to compile file into this library,
+                              if None, the VUNIT_VHDL_STANDARD environment variable is used
+        """
+        if vhdl_standard is None:
+            vhdl_standard = self._vhdl_standard
+        
+        libs_and_files = dict()
+
+        with open(project_csv_path) as f:
+            content = csv.reader(f)
+            for row in content:
+                if len(row) == 2: 
+                    lib_name = row[0].strip()
+                    file_name = row[1].strip()
```

Comment:
It would be reasonable to interpret the file name as relative to the location of the .csv file or an absolute path rather than relative to the current working directory when this code is executing,

Reply:
What I mean with the file name in this context is the file name that is written inside the csv file. 

For example if a have the following structure:
/directory/proj.csv
/directory/file.vhd
/run.py

If proj.csv contains "lib, file.vhd" and I run "python run.py" in the root directory it should refer to /directory/proj.csv making the semantics of the proj.csv file independent on the current working directory where the run.py script is executed.

Reply:
I understand what you want me to do, I just don't know how, can you please explain a little more?

Reply:
You can achieve what I describe by. 
```python
from os.path import normpath, join, dirname
normpath(join(dirname(csv_file_name), file_name))

It will make file_name relative to the parent directory of the csv file unless file name is an absolute path already.
```